CalibCalorimetry/EcalTrivialCondModules/src/EcalTrivialConditionRetriever.cc

```diff
@@ -27,7 +27,8 @@
 using namespace edm;
 
 EcalTrivialConditionRetriever::EcalTrivialConditionRetriever(const edm::ParameterSet& ps) {
-  std::string path = "CalibCalorimetry/EcalTrivialCondModules/data/";
+  //  std::string path = "CalibCalorimetry/EcalTrivialCondModules/data/";
+  std::string path = "CalibCalorimetry/EcalTrivialCondModules/data_test/";
```

Comment:
If I have a correct understanding.
There could be some effect when other user tries to use this package.
Will it be better to change the path with an optional configuration?

Reply:
Could you suggest stgh with this optional argument ?  Thanks.

Reply:
Unless there are some special concern, `ps.getUntrackedParameter<std::string>("dataPath", "CalibCalorimetry/EcalTrivialCondModules/data/");` should be fine, with which the default path will be taken when `dataPath` is not explicitly defined in the corresponding Python configuration file. 

Ref: The path of this format `CalibCalorimetry/EcalTrivialCondModules/data/` is special in CMSSW, as described in [twiki](https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideBuildFile#CmsswData).